src/wp-admin/js/editor-expand.js

```diff
@@ -110,50 +150,77 @@
 				sideSortablesHeight: $sideSortables.height() || 0
 			};
 
-			// Adjust for hidden
+			// Adjust for hidden menubar.
 			if ( heights.menuBarHeight < 3 ) {
 				heights.menuBarHeight = 0;
 			}
 		}
 
 		// We need to wait for TinyMCE to initialize.
+		/**
+		 * @summary Binds all necessary functions for editor expand to the editor,
+		 * when the editor is initialized.
+		 *
+		 * @since 4.0.0
+		 *
+		 * @param {event} event The TinyMCE editor init event
+		 * @param {object} editor The editor to bind the vents on.
+		 *
+		 * @returns
+		 */
 		$document.on( 'tinymce-editor-init.editor-expand', function( event, editor ) {
+			// VK contains the type of key pressed. VK = virtual keyboard.
 			var VK = window.tinymce.util.VK,
+				/**
+				 * @summary Hides any float panel with a hover state, and tooltips.
```

Comment:
I'd consider rewriting this to something like: `Hides any float panel with a hover state. Additionally hides tooltips.`.